db_router.py

```diff
@@ -0,0 +1,34 @@
+
+
+class TmpBaseRouter(object):
+    """
+    A router to control all database operations on models in the
+    auth application.
+    """
+    def db_for_read(self, model, **hints):
+        """
+        Attempts to read auth models go to auth_db.
+        """
+        if model._meta.model_name == 'search':
+            return 'tmp'
+        return None
+
+    def db_for_write(self, model, **hints):
+        """
+        Attempts to write auth models go to auth_db.
+        """
+        if model._meta.model_name == 'search':
+            return 'tmp'
+        return None
+
+    def allow_relation(self, obj1, obj2, **hints):
+        """
+        Allow relations if a model in the auth app is involved.
+        """
+        if obj1._meta.model_name == 'search' or \
+           obj2._meta.model_name == 'search':
+            return True
+        return None
+
+    def allow_migrate(self, db, app_label, model_name=None, **hints):
```

Comment:
I think we should keep migrations up: when Search models (and later catalog) will change, we need a way to regenerate the db. We decided in #335 to specifically use a migration for that instead of adding a step in the Debian install/update process. I think we should follow that way.